org.eclipse.winery.frontends/app/tosca-management/src/app/instance/sharedComponents/interfaces/interfaces.component.ts

```diff
@@ -350,6 +413,12 @@ export class InterfacesComponent implements OnInit {
         this.loading = false;
     }
 
+    private handleInheritedInterfaceData(data: InheritedInterface[]) {
+        console.log(data);
+        this.inheritedInterfacesData = data ? data : [];
+        this.loading = false;
```

Comment:
You should only use one kind of defining that the component is currently loading - either your new one or the old one.